### DISCUSSION TITLE
Moving in_EAX (and others parameter passed in register) to function operands

### DISCUSSION THREAD

**Discussion by SerVB** [NationalSecurityAgency/ghidra#3056] [Q&A]
I see that IDA Pro adds parameters to the function declaration after visiting this function if these parameters are passed in registers.

However, I can't find a way to make Ghidra show these arguments in the declaration. Is there any?

**Comment by astrelsky** [comment]
Right click on the function, select edit function signature. Set the calling convention to the correct one. If it is not present, or doesn't properly follow a convention, check the "use custom storage " and assign the storage as necessary.
